xterm/index.cgi

```diff
@@ -175,6 +175,13 @@ my $user = $access{'user'};
 if ($user eq "*") {
 	$user = $remote_user;
 	}
+elsif ($user eq "root" && $remote_user ne $user && !$in{'user'}) {
+	# If possible, start with a sudo-capable user
+	my @uinfo = getpwnam($remote_user);
+	if (defined($uinfo[0]) && $uinfo[7]) {
```

Comment:
Change `defined($uinfo[0])` to just `@uinfo`

Reply:
Done!